src/client/opamCommands.ml

```diff
@@ -1181,10 +1181,22 @@ let env =
     mk_flag ["revert"]
       "Output the environment with updates done by opam reverted instead."
   in
+  let status =
+    mk_flag ["status"]
+      "Fail if environment is not synchronised. In that case display not upt to date variables"
```

Comment:
```suggestion
      "Exits with 0 if the environment is already up-to-date, 1 otherwise, after \
       printing the list of not up-to-date variables."
```

Reply:
I would also rename the option with `--check`, since it is a bit similar to `opam update --check`.